model/scikit/LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright (c) 2017-2019 nucleon
```

Comment:
Who is `nucleon`?

Reply:
Since this is going into the main repo, it's all under Intel, similarly to

https://github.com/google/oss-fuzz/blob/master/projects/boost/build.sh#L2

So this should be

```
Copyright (c) 2019 Intel
```